packages/flutter_tools/templates/skeleton/pubspec.yaml.tmpl

```diff
@@ -0,0 +1,32 @@
+name: {{projectName}}
+description: {{description}}
+
+# Prevent accidental publishing to pub.dev.
+publish_to: 'none'
+
+version: 1.0.0+1
+
+environment:
+  sdk: {{dartSdkVersionBounds}}
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_localizations:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+  flutter_lints: ^1.0.0
+
+flutter:
+  uses-material-design: true
+
+  # Enable generation of localized Strings from arb files.
+  generate: true
+
+  assets:
+    # Add assets from the images directory to the application.
+    - assets/images/
```

Comment:
there are a lot fewer comments in this file than in the main template's pubspec.yaml, is that intended?

Reply:
This is intentional: I thought it's okay since this template is focused on intermediate users who "just want to start an app". But I don't have a strong opinion here, and can easily revert back. Let me know.

> Aside: When the comments are back, 95% of the lines in this file are the same as in the main template (we're just actually using `assets:` here). I was looking at ways to make this DRY, but it would just make bind this new template to `create_base.dart` with something like [`withPluginHook`](https://github.com/flutter/flutter/blob/34059eec2c671472ee95f5093705fb9fed0b3919/packages/flutter_tools/lib/src/commands/create_base.dart#L368).